### ISSUE TITLE
Standarization of certain terms

### ISSUE THREAD

**Issue by gustavovelascoh** [shervinea/cheatsheet-translation#30]
For example, there is not a word in Spanish for "cheatsheet". There exist some words but they are used by certain regions, (In my opinion). I was thinking of translations as "Hoja de ayuda" but I found a translator used "Hoja de trucos", which seems better to me.

Like "cheatsheet", there are other terms/words that don't have a simple translation and 2 translator could use different words. I propose to standardize some expressions looking for the consistency across all the material.

**Comment by fdiazgon** [comment]
I agree. Maybe we can take a look at the [French translation](https://github.com/shervinea/cheatsheet-translation/tree/master/fr) done by the authors. For example, we can see that neither "Dropout" or "Cross-entropy" are translated (to mention some).

**Comment by jpchavat** [comment]
For 'cheatsheet' -> "Hoja de referencia"
Please, vote to agree with the term.
